### PR TITLE
language fixes

### PR DESCRIPTION
**Pull request by EugeneJoe**

### CODE REVIEW

File: backend/passport-config.js [hunk 1]
```diff
@@ -67,12 +67,14 @@ const { MongoClient } = require("mongodb");
           const user = await getUserByUsername(username);
 
           if (!user) {
+            console.log('Incorrect username');
```
Comment (yonasbek) [inline backend/passport-config.js:70]: console log on passport should be removed

File: backend/routes/index.js [hunk 2]
```diff
@@ -975,8 +975,7 @@ module.exports = function() {
   router.get("/notifications/byUser", (req, res) => {
     try {
       MongoClient.connect(dbUrl, (err, db) => {
-        let dbo = db.db(dbFolder);
-        // TODO: Fix this when eugene creates a new notification table so get that information to from that table
```
Comment (yonasbek) [inline backend/routes/index.js:979]: reminder for self lol. Should be removed

File: sb.web.app/package.json [hunk 3]
```diff
@@ -59,7 +60,8 @@
     "eslint-plugin-import": "^2.20.2",
     "eslint-plugin-node": "^1.0.0",
     "eslint-plugin-promise": "^4.2.1",
-    "eslint-plugin-standard": "^4.0.0"
+    "eslint-plugin-standard": "^4.0.0",
+    "eslint-plugin-vue": "^9.17.0"
```
Comment (yonasbek) [inline sb.web.app/package.json:64]: is this a necessary package

Comment (EugeneJoe) [reply sb.web.app/package.json:64]: I had to install it cause the code wasn't running without it.
Might have been something that went wrong just on my local.

**Review by yonasbek — APPROVED** [review]

**Comment by yonasbek** [comment]
LGTM

**Pull request merged**